modules/core/28-wasm/keeper/keeper.go

```diff
@@ -60,52 +55,37 @@ func NewKeeper(cdc codec.BinaryCodec, key sdk.StoreKey, vmConfig *VMConfig, vali
 	}
 }
 
-func (k Keeper) PushNewWASMCode(ctx sdk.Context, clientType string, code []byte) ([]byte, string, error) {
+func (k Keeper) PushNewWasmCode(ctx sdk.Context, code []byte) ([]byte, error) {
 	store := ctx.KVStore(k.storeKey)
-	codeHash := generateWASMCodeHash(code)
+	codeHash := generateWasmCodeHash(code)
+	codeIDKey := types.CodeID(codeHash)
 
-	latestVersionKey := types.LatestWASMCode(clientType)
+	if store.Has(codeIDKey) {
```

Comment:
You will remove this later. I had this in an old version of cosmwasm for consistency and got all kinds of complaints. I can this will happen here as well.